src/components/DropdownHeader.js

```diff
@@ -10,11 +10,15 @@ import { QUERY_TYPES } from '../queries';
 import { DropdownSelect } from './DropdownSelect';
 import { Wrapper } from './Wrapper';
 
-const dropdownEntries = (query, queryVariables, data, excludedDataProviders) => {
+const dropdownData = [];
+
+const dropdownEntries = (query, queryVariables, data, excludedDataProviders, isLocationFilter) => {
   // check if there is something set in the certain `queryVariables`
   // if not, - Alle - will be selected in the `dropdownData`
   const selected = {
-    [QUERY_TYPES.EVENT_RECORDS]: !queryVariables.categoryId,
+    [QUERY_TYPES.EVENT_RECORDS]: isLocationFilter
+      ? !queryVariables.location
+      : !queryVariables.categoryId,
```

Comment:
Unfortunately I don't think it will work, because we have two different filters and 4 different options.

The user can apply no filter at all,
the user can only apply a location filter,
the user can only apply a category filter,
the user can apply both filters.

We cannot control all of these options in an interdependent manner. We need to maintain the condition that one is selected while the other is deselected. Otherwise, after one filter is selected, the other filter is automatically selected, but without any value in the dropdown.